### DISCUSSION TITLE
public facing gallery?

### DISCUSSION THREAD

**Discussion by khord** [immich-app/immich#328] [Q&A] [answered]
Before I decide if I want to try immich out, I wanted to inquire whether or not there is the ability to have a public facing gallery/album where no sign in is required. It is not clear from the screenshots whether this is possible.

My use case is that I want to be able to sync / upload photos from my phone, and have a public site that I can give to anyone to view a photo feed.

**Comment by alextran1502** [comment, marked as answer]
There is no public facing instance, similar to Google Photos.

**Comment by Danie10** [comment]
Just wanted to mention I'd also like to have this as I share quite a few hiking trips etc on social media and its nice to link back to a publicly available album. I realise we have the link share per album, but no-oner is able to see a list, or browse, our other publicly shared albums from the site.
